apiary.apib

```diff
@@ -1682,12 +1682,476 @@ root will not be listable, as it is virtual. Access known paths directly.
 ## stat
 
 # Group pin
+Pin (and unpin) objects to local storage
 
-## add
+This command can't be called directly.
 
-## ls
+## add [GET /pin/add{?arg}{&recursive}]
+Pins objects to local storage
+
+Retrieves the object named by <ipfs-path> and stores it locally
+on disk.
+
++ Parameters
+    + arg (string, required) - Path to object or objects to be pinned
+    + recursive (boolean, optional) - Recursively pin the object linked to by the specified object or objects
```

Comment:
Not seeing this in the request on wireshark after calling `curl -i -X POST "http://localhost:5001/api/v0/pin/add?arg=QmW2WQi7j6c7UgJTarActp7tDNikE4B2qXtFCfLPdsgaTQ"`